CalendarApp/BookingRoom/Controllers/DeleteController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using BookingRoom.Models;
+using BookingRoom.Models.GoogleConnection;
+using log4net;
+
+namespace BookingRoom.Controllers
+{
+    public class DeleteController : EventController
```

Comment:
I think it is time to separate existing logic into business and communication layer. It would be good to move all business related logic into service that can be reused and in communication layer leave only service calls